sdk/identity/identity/test/internal/node/defaultAzureCredential.spec.ts

```diff
@@ -26,6 +26,23 @@ describe("DefaultAzureCredential", () => {
     process.env.AZURE_TOKEN_CREDENTIALS = "dev";
     expect(() => new DefaultAzureCredential()).not.toThrowError();
   });
+
+  it("should throw if requiredEnvVar is missing", () => {
```

Comment:
I noticed `requiredEnvVar` is singular here, but it's plural down below. Can you really specify more than one env var (e.g., a comma-delimited list)?

Reply:
We could specify multiple env variables, if we extend the type to -
`requiredEnvVars?: DefaultEnvVars | DefaultEnvVars[];` 

and export type DefaultEnvVars = "AZURE_TOKEN_CREDENTIALS" | "AZURE_CLIENT_ID"